### ISSUE TITLE
Import of vis-2 project needs to be done twice

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.vis-2#97]
**Describe the bug**  

**To Reproduce**  
1. Export an existing vis-2 project (based on vis-2 version 2.3.0 with vis2 compatible widgets)
2. Import the zip file of the exported project by using dialog in vis-2 editor
![grafik](https://github.com/ioBroker/ioBroker.vis-2/assets/25422320/3ae73e31-2b45-4307-b97f-f6f99e0f2fe3)

3. confirm diaglog to open project after import
4. Screen will be show
![grafik](https://github.com/ioBroker/ioBroker.vis-2/assets/25422320/4aee2cfd-4f76-4b2f-bb98-ac587aa65bd2)
5. Click on project "Resp_Haus2"
6. Project will not load (see log file)

The import is successfull after importing the zip file in dialog which is shown under step 4. once again. After this import the project will open successfully

**Screenshots & Logfiles**  
```
iobroker.vis-2@2.3.0 using theme "colored" [bootstrap.jsx:16:8](http://192.168.178.54:8082/vis-2/static/js/bootstrap.jsx)

██╗ ██████╗ ██████╗ ██████╗  ██████╗ ██╗  ██╗███████╗██████╗ 
██║██╔═══██╗██╔══██╗██╔══██╗██╔═══██╗██║ ██╔╝██╔════╝██╔══██╗
██║██║   ██║██████╔╝██████╔╝██║   ██║█████╔╝ █████╗  ██████╔╝
██║██║   ██║██╔══██╗██╔══██╗██║   ██║██╔═██╗ ██╔══╝  ██╔══██╗
██║╚██████╔╝██████╔╝██║  ██║╚██████╔╝██║  ██╗███████╗██║  ██║
╚═╝ ╚═════╝ ╚═════╝ ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝
[Prompt.js:14:12](http://192.168.178.54:8082/vis-2/static/node_modules/@iobroker/adapter-react-v5/Prompt.js)
Nice to see you here! :) Join our dev community here https://github.com/ioBroker/ioBroker or here https://github.com/iobroker-community-adapters [Prompt.js:15:12](http://192.168.178.54:8082/vis-2/static/node_modules/@iobroker/adapter-react-v5/Prompt.js)
Help us to create open source project with reactJS! [Prompt.js:16:12](http://192.168.178.54:8082/vis-2/static/node_modules/@iobroker/adapter-react-v5/Prompt.js)
See you :) [Prompt.js:17:12](http://192.168.178.54:8082/vis-2/static/node_modules/@iobroker/adapter-react-v5/Prompt.js)
[2023-10-02T19:28:37.035Z] Try to connect [socket.io.js:66:41](http://192.168.178.54:8082/lib/js/socket.io.js)
Cannot read project file "Resp_Haus2/vis-views.json": Not exists
```

**Versions:**  
 - Adapter version: vis-2 version 2.3.0

**Comment by PiemurIsBack** [comment]
Das Verhalten hatte ich auch.
Nachdem ich JS-Controller version: 5.0.13 installiert habe, konnte ich Projekte wieder importieren.
In meinem Fall das Demo-Projekt MD_Demo von Uhula.

Siehe auch https://github.com/ioBroker/ioBroker.vis-2/issues/78
JS-Controller version: 5.0.13 Damit funktioniert es.


Versions:
Adapter version: v2.3.0
JS-Controller version: 5.0.13
Node version: v18.17.0
Operating system: rapsy 64bit lite bullseye

**Comment by foxriver76** [comment]
Can you reevaluate this with current controller @Feuer-sturm

**Comment by Feuer-sturm** [comment]
@foxriver76 Issue is fixed with js-controllre 5.0.15
Thank you :-)
